guide/src/SUMMARY.md

```diff
@@ -0,0 +1,25 @@
+# Summary
+
+[Introduction](README.md)
+
+- [Installation and Usage](installation_and_usage.md)
+- [Configuration](configuration.md)
+- [Clippy's Lints](lints/README.md)
+    - [Cargo]()
+    - [Complexity]()
+    - [Correctness]()
+    - [Pedantic]()
+    - [Perf]()
+    - [Restriction]()
+    - [Style]()
+    - [Deprecated]()
+    - [Nursery]()
+- [Development](development/README.md)
+    - [Basics](development/basics.md)
+    - [Adding Lints](development/adding_lints.md)
+    - [Common Tools](development/common_tools_writing_lints.md)
+    - [Backporting Changes](development/backport.md)
+    - [Updating the Changelog](development/changelog_update.md)
+    - [Release a New Version](development/release.md)
```

Comment:
It may make sense to put these in their own category names "Infrastructure", since normal contributors usually don't have to interact with those.

Reply:
done